engine/api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ProcessManager.scala

```diff
@@ -9,7 +9,11 @@ import scala.concurrent.Future
 trait ProcessManager {
 
   //TODO: savepointPath is very flink specific, how can we handle that differently?
-  def deploy(processId: ProcessVersion, processDeploymentData: ProcessDeploymentData, savepointPath: Option[String]) : Future[Unit]
+  def deploy(processId: ProcessVersion, processDeploymentData: ProcessDeploymentData, savepointPath: Option[String], user: String) : Future[Unit]
```

Comment:
WDYT about using value objects pattern? Naming something "user" without adding name or id is the worst option ... ;-)

Reply:
I added User object with id and name :)